src/css/base.css

```diff
@@ -0,0 +1,36 @@
+@import url('theme.css');
```

Comment:
the build should compile all this into a single file, how does the `@import` works in this case with webpack and rollup ?

Reply:
Well looks like it works for webpack, not sure about rollup: https://github.com/webpack-contrib/css-loader#import

Reply:
can you test this on rollup build ?

Reply:
what command do I need to run to test this build?

Reply:
I know webpack supports this syntax:
```
@import url('theme.css')  and @import 'theme.css'
```
Can you confirm rollup also support it ? the command is `yarn build`

Reply:
Tested with rollup, breaks the build if i use @import inside a css file, imported all relevant css files in index.js using import './css/<css-name>'

Reply:
Ok, too bad `rollup` don't support that syntax :disappointed: